integration-test/src/step_definitions/support/steps.js

```diff
@@ -30,6 +30,8 @@ const { createOrganizationInfo, createServiceInfo, sendInvalidDemandPaymentNotic
 const { gpdSessionBundle, gpsSessionBundle } = require('./utility/data');
 const { getValidBundle } = require('./utility/helpers');
 
+// increase cucumber promise timeout
+setDefaultTimeout(15000);
```

Comment:
why ? it take more than [5sec](https://github.com/cucumber/cucumber-js/blob/main/docs/support_files/timeouts.md) ?

Reply:
yes @pasqualespica, at least when I pointed to the development database I noticed that it timed out in 5 seconds (it need to enable archiving in dev)